components/gc-srvinfo/index.json-ld

```diff
@@ -108,6 +108,61 @@
 					"fr": "Août 2014 - Implémentation initiale de la variante."
 				}
 			]
+		},
+		{
+			"name": {
+				"en": "Services and information with FieldFlow",
+				"fr": "Services et renseignements avec FieldFlow"
+			},
+			"status": "stable",
+			"description": {
+				"en": "Variation of the Service and Information component using FieldFlow.",
+				"fr": "Variante de la composante de Services and Renseignements qui utilise FieldFlow."
+			},
+			"guidance": {
+				"en": "https://design.canada.ca/common-design-patterns/services-information.html",
+				"fr": "https://conception.canada.ca/configurations-conception-communes/services-renseignements.html"
+			},
+			"iteration": "_:iteration_srvinfo_6",
+			"example": [
+				{
+					"en": { "href": "gc-srvinfo-en.html#eg1", "text": "3 columns pattern" },
+					"fr": { "href": "gc-srvinfo-fr.html#eg1", "text": "Disposition sur 3 colonnes" }
+				},
+				{
+					"en": { "href": "gc-srvinfo-en.html#eg2", "text": "2 columns pattern" },
+					"fr": { "href": "gc-srvinfo-fr.html#eg2", "text": "Disposition sur 2 colonnes" }
+				},
+				{
+					"en": { "href": "gc-srvinfo-en.html#eg3", "text": "Doormat links in single column pattern" },
+					"fr": { "href": "gc-srvinfo-fr.html#eg3", "text": "Disposition sur 1 colonne" }
+				},
+				{
+					"en": { "href": "gc-srvinfo-en.html#eg4", "text": "Sequential doormat links in single column pattern" },
+					"fr": { "href": "gc-srvinfo-fr.html#eg4", "text": "Liens séquentiels avec disposition sur 1 colonne" }
+				},
+				{
+					"en": { "href": "gc-srvinfo-en.html#eg5", "text": "Doormat links in 3 columns grouped with headings pattern" },
+					"fr": { "href": "gc-srvinfo-fr.html#eg5", "text": "Disposition groupée avec titres, sur 3 colonnes" }
+				},
+				{
+					"en": { "href": "gc-srvinfo-en.html#eg6", "text": "3 columns pattern with invisible header" },
+					"fr": { "href": "gc-srvinfo-fr.html#eg6", "text": "Disposition sur 3 colonnes avec en-tête invisible" }
+				},
+				{
+					"en": { "href": "gc-srvinfo-en.html#eg7", "text": "FieldFlow drop-down list" },
+					"fr": { "href": "gc-srvinfo-fr.html#eg7", "text": "Liste déroulante FieldFlow" }
+				}
```

Comment:
Add only the link to the new list of working examples mentioned in my main comment.